### PR TITLE
Orthogonal rewards

### PR DESCRIPTION
**Pull request by Samia1117**

### CODE REVIEW

File: movie-review-finetuning/finetune_gpt2.py [hunk 1]
```diff
@@ -167,14 +150,19 @@ def collator(data):
...
                 for output in pipe_outputs
                 for item in output
                 if item["label"] == "POSITIVE"
             ]
-            rewards = [torch.tensor(score) for score in positive_scores]
+
+            rewards = []
+            for i in range(len(positive_scores)):
+                print(f'response_tensors[i] = {response_tensors[i]}')
+                score = (0.5 * positive_scores[i]) + (0.5 * len(response_tensors[i]))
```
Comment (Samia1117) [inline movie-review-finetuning/finetune_gpt2.py:164]: There's a bug here. We need to normalize the two rewards first. (The range of positive scores and range of length of response tensors will be very different)

File: movie-review-finetuning/finetune_gpt2.py [hunk 2]
```diff
@@ -167,14 +150,19 @@ def collator(data):
             texts = [q + r for q, r in zip(batch["query"], batch["response"])]
             pipe_outputs = sentiment_pipe(texts, **sent_kwargs)
             
-            # TODO: Change this for negative/neutral review based fine tuning
             positive_scores = [
-                item["score"]
+                # Assign LOW SCORES to 'positiveness' of the generated text
+                -1 * item["score"]
```
Comment (Samia1117) [inline movie-review-finetuning/finetune_gpt2.py:155]: Bug #2 !!

File: movie-review-finetuning/finetune_gpt2.py (hunk 1 above)
Comment (Samia1117) [reply movie-review-finetuning/finetune_gpt2.py:164]: Look into 
- CLIP ing [Maybe something like: min(4, len(response)]
- KL Divergence
